drivers/gl/glyph_page.go

```diff
@@ -136,25 +96,10 @@ func (p *glyphPage) add(rune rune, g *glyph) bool {
 		return false // Page full
 	}
 
-	// Build the raster contours
-	p.rast.Clear()
-	fx := -raster.Fix32((int64(g.B.XMin) * int64(p.resolution)) >> 14)
-	fy := +raster.Fix32((int64(g.B.YMax) * int64(p.resolution)) >> 14)
-	e0 := 0
-	for _, e1 := range g.End {
-		p.drawContour(g.Point[e0:e1], fx, fy)
-		e0 = e1
-	}
-
-	// Perform the rasterization
-	a := &image.Alpha{
-		Pix:    p.image.Pix[x+y*p.image.Stride:],
-		Stride: p.image.Stride,
-		Rect:   image.Rect(0, 0, w, h),
-	}
-	p.rast.Rasterize(raster.NewAlphaSrcPainter(a))
+	_, _, mask, maskp, _ := face.Glyph(fixed.Point26_6{}, r)
```

Comment:
BTW, you'll need to make another (simple) change for
https://github.com/golang/freetype/commit/03bdcc0fe1c41fb81c28f5ad739241c1f0b6ec9b

_, _, mask, maskp, _
would become
_, mask, maskp, _, _

Sorry.